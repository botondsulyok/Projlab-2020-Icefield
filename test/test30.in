field spawn 0
entity esk1 Eskimo spawn
entity esk2 Eskimo spawn
item tent1 Tent esk1
stats spawn
stats esk1
stats esk2
tent esk1
stats spawn
stats esk1
stats esk2
storm esk1
stats esk1
stats esk2
storm esk2
stats spawn
stats esk1
stats esk2