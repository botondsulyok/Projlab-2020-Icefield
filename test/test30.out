id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: esk1, esk2
pullers: esk1, esk2
masters: esk1, esk2
neighbours: null

id: esk1
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent1
components: null

id: esk2
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: spawn
type: iceunit
building: tent1
snow: 0
pickable: null
standers: esk1, esk2
pullers: esk1, esk2
masters: esk1, esk2
neighbours: null

id: esk1
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent1
components: null

id: esk2
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: esk1
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent1
components: null

id: esk2
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: spawn
type: iceunit
building: tent1
snow: 0
pickable: null
standers: esk1, esk2
pullers: esk1, esk2
masters: esk1, esk2
neighbours: null

id: esk1
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent1
components: null

id: esk2
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 4
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

